lighthouse/config/defaults.py

```diff
@@ -27,7 +28,41 @@
 ALLOW_UNKNOWN = True
 DEBUG = True
 HATEOAS = True
-DOMAIN: Dict = {"samples": {}, "imports": {}, "centres": {}, "schema": {}}
+
+SAMPLES_DECLARATIONS_SCHEMA: Dict = {
+    "root_sample_id": {
+        "type": "string", 
+        "required": True,
+        "validation_errors" : True
+    },
+    "value_in_sequencing": {
+        "type": "string",
+        "allowed": ["Yes", "No", "Unknown"],
+        "required": True,
+    },
+    "declared_at": {
+        "type": "datetime",
+        "required": True,
+    },
+}
+# We are overwriting the date format. 
+# By default eve DATE_FORMAT is set to RFC1123 standard which is %a, %d %b %Y %H:%M:%S GMT
+
+DATE_FORMAT = r"%Y-%m-%dT%H:%M:%S"
+# eg "2013-04-04T10:29:13"
+
+DOMAIN: Dict = {
+    "samples": {"resource_methods": ["GET", "POST"], "bulk_enabled": True},
+    "imports": {},
+    "centres": {},
+    "samples_declarations": {
+        "resource_methods": ["GET", "POST"],
+        "item_methods": ["GET", "PUT", "DELETE"],
```

Comment:
Whats the item_methods for?

Reply:
Access to GET PUT and DELETE single declaration instances. Not needed, so I'll remove